scripts/checkApiUpToDate.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# NOTE: there is no way to extract a single file from github using the git interface.
+# However, github provides a subversion interface and that does let you grab a file.
+# So we first test to see if svn is available. If not, we fail. If so, we use it to grab
+# the file and do the compare.
+svnimage=$(which svn)
+if [ -z "$svnimage" ]; then
+  echo "No svn so cannot retrieve the api yaml file"
+  exit 2
+fi
+
+tmpfile=/tmp/data-repository-openapi.yaml
+svn export --force --quiet \
+  https://github.com/DataBiosphere/jade-data-repo/trunk/src/main/resources/data-repository-openapi.yaml \
```

Comment:
You could `curl` this instead: https://raw.githubusercontent.com/DataBiosphere/jade-data-repo/develop/src/main/resources/data-repository-openapi.yaml

I think curl will be more likely to be on someone's system.